lecture_notes/10 refactoring/3-map.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+function shift(offset, points) {
+    points.forEach(function (point) {
+        point.x += offset.x;
+        point.y += offset.y;
+    });
+    return points;
+}
+
+function parser(point){
+    if(typeof(point) == 'string')
```

Comment:
```suggestion
    if (typeof point === 'string')
```
- Typeof is not an operator
- Don't use `==`